ts/src/pro/poloniex.ts

```diff
@@ -0,0 +1,965 @@
+//  ---------------------------------------------------------------------------
+
+import poloniexRest from '../poloniex.js';
+import { BadRequest, AuthenticationError } from '../base/errors.js';
+import { ArrayCache, ArrayCacheByTimestamp, ArrayCacheBySymbolById } from '../base/ws/Cache.js';
+import { Precise } from '../base/Precise.js';
+import { sha256 } from '../static_dependencies/noble-hashes/sha256.js';
+
+//  ---------------------------------------------------------------------------
+
+// @ts-expect-error
+export default class poloniex extends poloniexRest {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'has': {
+                'ws': true,
+                'watchOHLCV': true,
+                'watchOrderBook': true,
+                'watchTicker': true,
+                'watchTickers': true,
+                'watchTrades': true,
+                'watchBalance': true,
+                'watchStatus': false,
+                'watchOrders': true,
+                'watchMyTrades': false,
+            },
+            'urls': {
+                'api': {
+                    'ws': {
+                        'public': 'wss://ws.poloniex.com/ws/public',
+                        'private': 'wss://ws.poloniex.com/ws/private',
+                    },
+                },
+            },
+            'options': {
+                'tradesLimit': 1000,
+                'ordersLimit': 1000,
+                'OHLCVLimit': 1000,
+                'watchOrderBookMethod': 'book_lv2', // can also be 'book'
```

Comment:
This is not the default way of doing `watchOrderBook: {"name":X}` should be enough